BPOR/NIHR.Infrastructure/Settings/AwsSecretsManagerSettings.cs

```diff
@@ -7,8 +8,8 @@ public class AwsSecretsManagerSettings : IValidatableObject
     {
         public static string SectionName => "AwsSecretsManagerSettings";
         public bool Enabled { get; set; } = true;
-        public string Region { get; set; } = string.Empty;
-        public string SecretName { get; set; } = string.Empty;
+        public string Region { get; set; } = Environment.GetEnvironmentVariable("AwsSecretsManagerSettings__Region");
```

Comment:
We shouldn't hard code the Environment dependency here.

This class is simply a container for the configuration, where the values come from should be handled in Startup.cs.